@()

@main("About") {
    
<p>
This work was supported by a grant from the <a href="http://www.snf.ch">Swiss National Science Foundation</a> and the <a href="http://www.grandchallenges.org/Pages/Default.aspx">Bill and Melinda Gates Foundation Grand Challenges Exploration</a>. 
</p>
<p>
This work was performed by 
<ul>
<li>
Pejman Mohammadi and Niko Beerenwinkel from the <a href="http://www.bsse.ethz.ch/cbg/people">Department of Biosystems Science and Engineering, ETH Zurich</a>. 
</li>
<li>
Beda Joos, Huldrych G&#252;nthard from the <a href="http://www.en.infektiologie.usz.ch/Pages/default.aspx" >Division of Infectious Diseases and Hospital Epidemiology, University Hospital Zurich</a>.
</li>
<li>
S&eacute;bastien Desfarges, Istvan Bartha, Nadine Zangger, Miguel Munoz, Amalio Telenti, Angela Ciuffi from the <a href="http://www.chuv.ch/imul">Institute of Microbiology, University Hospital Center and University of Lausanne</a>.
</li>
</ul>
</p>

<p> The following public databases were used in this analysis: 

	<ol>
	<li>	Location-specific: Genes were labeled according to two separate types of co-localization, one classifying genes by their physical position on the chromosomal bands, and the other according to the Gene Ontology (GO) cellular component classification of the genes. Both annotations are available from the Molecular Signatures Database (MSigDB ver. 3, www.broadinstitute.org/gsea/msigdb) (<a href="http://www.pnas.org/content/102/43/15545.full">A. Subramanian 2005</a>).
	</li>

<li>	Sequence-based: We checked sequence-based regulations by analyzing sets of genes that share the same transcription factor binding motif as defined in the TRANSFAC database (version 7.4, http://www.gene-regulation.com), as well as those sharing 3â€™UTR miRNA binding site as reported in (<a href="http://www.ncbi.nlm.nih.gov/pubmed/15735639">X. Xie 2005</a>).</li>

<li>Functional: We used canonical pathway classification of genes according to the Reactome database (ver. 40, http://www.reactome.org), GO biological process, GO molecular function, and a selected list of canonical pathways included in MSigDB from KEGG pathways (www.genome.jp/kegg/pathway.html) and BioCarta (www.biocarta.com/genes/index.asp). </li>

<li>HIV-related: We compiled a list of previously reported HIV-1 related genes. This list included HIV-1 host factors reported in (<a href="http://www.ncbi.nlm.nih.gov/pubmed/18854154">R. Konig 2008</a>, <a href="http://www.sciencemag.org/content/319/5865/921.full">A. L. Brass 2008</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/19460752">M. L. Yeung 2009</a>,<a href="http://f1000.com/1127006">H. Zhou 2008</a>) as well as the genes classified by the viral protein-protein interaction partner of their corresponding protein product, reported for each viral protein in (<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature10719.html%3FWT.ec_id%3DNATURE-20111222">S. Jager 2012</a>) and in the VirusMINT online database (http://mint.bio.uniroma2.it/virusmint).</li>
<li>TarBase: The <a href="http://www.microrna.gr/tarbase">TarBase 6.0</a> database was used in this study. <a href="http://nar.oxfordjournals.org/content/40/D1/D222.abstract">Vergoulis et al. 2012</a> 
</ol>
</p>

<p> Commercial use of this resource is prohibited. </p>
 
 <small>A recent web browser is required to view the web pages (Internet Explorer 9, any version of Safari, Firefox, Chrome, Opera).</small>
    
}